docs/false_positives.md

```diff
@@ -0,0 +1,243 @@
+Table of Contents
+=================
+* [False positives](#false-positives)
+  * [What target to analyze?](#what-target-to-analyze)
+  * [Infeasible path](#infeasible-path)
+    * [Correlated conditions](#correlated-conditions)
+    * [Partial functions](#partial-functions)
+    * [Prefer standard functions](#prefer-standard-functions)
+    * [Do not turn off code checks](#do-not-turn-off-code-checks)
+    * [Suppress specific dead store warnings](#suppress-specific-dead-store-warnings)
+    * [Alternative implementations](#alternative-implementations)
+  * [Syntax based checks](#syntax-based-checks)
+  * [Suppress results](#suppress-results)
+    * [3rd party code](#third-party-code)
+    * [Authored code](#authored-code)
+    * [Unauthored code](#unauthored-code)
+
+## <a name="false-positives"></a> False positives
```

Comment:
This is basically **the** document itself? Why is it a `h2` instead of a `h1` then? This makes it semantically to be a "part" of the `Table of Contents` section, which it isn't.

Along with it, a `#` should be removed from each headings below, of course.